src/style/card.scss

```diff
@@ -0,0 +1,82 @@
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 944px;
+  margin: 0 auto;
+  margin-top: 50px;
+}
+
+.card {
+  height: 408px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-bottom: 46px;
+
+  &__logo {
+    height: 134px;
+    width: 160px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    margin: 40px 16px 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    margin: 4px 16px 0;
+    color: #616070;
+  }
```

Comment:
let's use paddings for card, should look this way
![Screenshot 2023-03-02 at 11 11 37](https://user-images.githubusercontent.com/75496546/222383801-890e0ba2-22b7-45f0-a26d-6ab2fe8ac9ea.png)